urc_perception/multiclass_segmentation/train.py

```diff
@@ -0,0 +1,147 @@
+import numpy as np
+import yaml
+import os
+
+# Import utilities for Torch
+import torch
+from torch.utils.data import DataLoader
+
+# Import model building and training utilities
+import segmentation_models_pytorch as smp
+from torch import optim
+import catalyst
+from catalyst import utils
+from torch.optim import AdamW
+from catalyst.dl import SupervisedRunner
+from catalyst.dl.callbacks import CriterionCallback, MulticlassDiceMetricCallback
+
+
+# Import dataset and data loaders
+from data_utils.segmentation_dataset import SegmentationDataset
+from data_utils.data_loaders import get_loaders
+from train_utils.helper_operations import CrossentropyND
+from train_utils.get_args import get_args
+from train_utils.save import save_result
+
+
+def main():
+    # Enable argument parsing for file paths
+    args = vars(get_args())
+
+    train_images_path = args["train_images"]
+    train_masks_path = args["train_masks"]
+    test_images_path = args["test_images"]
+    test_masks_path = args["test_masks"]
+
+    # print out yaml file configuration
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    yaml_path = os.path.join(dir_path, "config/igvc.yaml")
```

Comment:
This should be `urc.yaml`

Reply:
Just pushed a change removing the last few IGVC references, few must have slipped by me when I was looking the code over